CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
+* **2015-14-02**: implement sitemap generation
+* **2015-14-02**: [BC BREAK] Relax method visibility on "\Symfony\Cmf\Bundle\SeoBundle\SeoPresentation"
```

Comment:
I would say: "[BC BREAK] Changed method visibility from private to public of SeoPresentation#getSeoMetadata()"